src/decays/specializations/H/decay_H_to_AZ.inc

```diff
@@ -19,15 +19,27 @@ double CLASSNAME::get_partial_width<Higgs, Photon, ZBoson>(
       res *= alpha_em_0/alpha_em;
    }
 
+   std::string tag = field_as_string<Higgs>(in_idx) + "-" + field_as_string<Photon>(out1_idx) + "-" + field_as_string<ZBoson>(out2_idx);
```

Comment:
Can this `tag` be made `const`?

Reply:
Done in 19f276db24c013e1a8b0c92bb00ede1d2418e08c.

Reply:
Thanks!